contrib/subtree/git-subtree.sh

```diff
@@ -161,7 +162,7 @@ command="$1"
 shift
 
 case "$command" in
-add|merge|pull)
+add|merge|pull|map)
```

Comment:
pedantic nit, might be useful to keep these in alpha order

Reply:
I was assuming logical rather than alphabetical ordering, where at least in the docs the more arcane options go towards the end. Happy to make the internal bits alphabetical though.

Reply:
that's fair, it so happens the existing options are both logically & alphabetically ordered. Not a significant comment either way